traffic_ops/app/lib/API/Profile.pm

```diff
@@ -222,15 +227,25 @@ sub create {
 	if ($existing_desc) {
 		return $self->alert("a profile with the exact same description already exists.");
 	}
-
+	
 	my $cdn = $params->{cdn};
 	my $type = $params->{type};
+	my $routing_disabled = defined($params->{routingDisabled}) ? $params->{routingDisabled} : 0;
```

Comment:
@dangogh - is there a better way to do this? I can't imagine there isn't...

Reply:
not really when it's being used as dbix input..

btw,  @dg4prez, @mitchell852  unit tests fail on this -- check out https://builds.apache.org/job/incubator-trafficcontrol-PR-trafficops-test/263/

Reply:
I don't understand why it's failing.  Here's the test locally:

kabletown app $ git status
On branch routing_ignore
Your branch is up-to-date with 'origin/routing_ignore'.

nothing to commit, working tree clean
kabletown app $ db/admin.pl --env=test reset
Creating user: traffic_ops
Dropping database: to_test
DROP DATABASE IF EXISTS to_test;
CREATE DATABASE to_test OWNER traffic_ops;
Creating database tables.
blah blah blah db stuff

kabletown app $ prove -qrp t/api/1.2/profile.t
t/api/1.2/profile.t .. Subroutine TrafficOps::has redefined at /opt/traffic_ops/app/local/lib/perl5/Mojo/Base.pm line 38.
Subroutine TrafficOps::has redefined at /opt/traffic_ops/app/local/lib/perl5/Mojo/Base.pm line 38.
NOTICE:  truncate cascades to table "asn"
NOTICE:  truncate cascades to table "cachegroup_parameter"
NOTICE:  truncate cascades to table "server"
NOTICE:  truncate cascades to table "staticdnsentry"
NOTICE:  truncate cascades to table "deliveryservice_server"
NOTICE:  truncate cascades to table "hwinfo"
NOTICE:  truncate cascades to table "servercheck"
t/api/1.2/profile.t .. 16/? DBIx::Class::ResultSet::search_rs(): search( %condition ) is deprecated, use search( \%condition ) instead at /opt/traffic_ops/app/lib/API/Parameter.pm line 137
t/api/1.2/profile.t .. ok
All tests successful.
Files=1, Tests=99,  4 wallclock secs ( 0.03 usr  0.01 sys +  2.41 cusr  0.59 csys =  3.04 CPU)
Result: PASS

Reply:
you need to run All the tests with prove -qrp t

Test Summary Report
-------------------
t/api/1.2/cdn.t                               (Wstat: 512 Tests: 56 Failed: 2)
  Failed tests:  13, 18
  Non-zero exit status: 2
t/api/1.2/config_files_apache_traffic_server.t (Wstat: 256 Tests: 54 Failed: 1)
  Failed test:  50
  Non-zero exit status: 1
t/api/1.2/deliveryservice.t                   (Wstat: 512 Tests: 180 Failed: 2)
  Failed tests:  83, 173
  Non-zero exit status: 2
t/profile.t                                   (Wstat: 256 Tests: 12 Failed: 1)
  Failed test:  9
  Non-zero exit status: 1

Reply:
OK, tests should be fixed.  Waiting on results.